comit/src/network/protocols/orderbook.rs

```diff
@@ -292,6 +383,10 @@ impl Orderbook {
         Ok((order, identities.0, identities.1))
     }
 
+    pub fn take(&mut self, peer_id: &PeerId, order_id: OrderId) -> RequestId {
```

Comment:
This API is leaking details of its implementation.

Why does the caller have to know about the peer id of the order they want to take? The order ID should be enough.

Reply:
Fair point.

Reply:
Good Tobin, this is gone already.